README.md

```diff
@@ -1,24 +1,218 @@
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Database initialization
+<!-- PROJECT DESCRIPTION -->
 
-* How to run the test suite
+# 📖 Rent/Buy a Car App <a name="about-project"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+**Rent/Buy a Car App:** is a full stack web application with a react client and rails api that allows users to book reservations for renting or buying a car.
 
-* Deployment instructions
+=> [Link to the front-end react app](https://github.com/deniodev/car_reservation_front_end.git)
 
-* ...
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+ <summary>Client</summary>
+  <ul>
+    <li><a href="https://react.dev/">ReactJS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **The user logs in to the website, only by typing the username**
+- **The user can see links to: Cars, Reserve, My Reservations, Add Car, and Delete Car**
+- **See a list of cars**
+- **User can select the item and see details of the item**
+- **user clicks the "Add item" link in the navigation panel they can see a form for adding a new item.**
+- **When the user clicks the "Delete item" link in the navigation panel they can see a list of all items with title and "Delete" button.**
+- **When the user clicks the "My reservations" link in the navigation panel they can see a list of their reservations (with information about item name, date and city).**
+- **"Add item" and "Delete item" links are accessible only by users who are admins.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- <a href="https://dev--cozy-concha-614dd5.netlify.app">Carhub App Live</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- `React`
+- `Redux`
+- `Ruby`
+- `Rails`
+- `Code editor (VSCode, RubyMine)`
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/unorjikingsley/Reservation-API.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd my-project
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rspec spec
+```
```

Comment:
- [ ] When I run the tests according to the instructions, this happens:

![image](https://github.com/unorjikingsley/Reservation-API/assets/57421823/1caebc1d-c48a-44be-9099-7c4c52d689cf)

This is because your requests tests are rswag tests, not rspec tests.

## Solution

Just be more specific in your instructions:

```
rspec ./spec/models
```

Result:

![image](https://github.com/unorjikingsley/Reservation-API/assets/57421823/086841f3-2021-4b77-824e-063432255510)

That way, all your rspec tests will pass. 💪